modules/apps/site-navigation/site-navigation-menu-item-layout/src/main/java/com/liferay/site/navigation/menu/item/layout/internal/type/LayoutSiteNavigationMenuItemType.java

```diff
@@ -122,11 +122,11 @@ public boolean exportData(
 			return false;
 		}
 
-		if (!ArrayUtil.contains(
-				portletDataContext.getLayoutIds(), layout.getLayoutId())) {
-
-			return false;
-		}
+//		if (!ArrayUtil.contains(
+//				portletDataContext.getLayoutIds(), layout.getLayoutId())) {
+//
+//			return false;
+//		}
```

Comment:
Hi @wanderlast . The problem informated in LPP when import .lar in master is related with pages [publics and privates](https://liferay.slack.com/archives/CL9RTSZ52/p1651223400942559). The contional add in [LPS-125086](https://issues.liferay.com/browse/LPS-125086) and [LPS-150132](https://issues.liferay.com/browse/LPS-150132) broke import in master. If reset that modificated the import it works normally. For resolve the principal bug reported in this LPP is necessary add custom field in manifest.xml, more specific in tag <missing-reference>.

Reply:
Hi @murilostodolni,

So looking at this, there appears to be a potential problem. First, it looks like there's potential problems with LPS-125086: please see https://issues.liferay.com/browse/PTR-3148. From what I'm understanding, you made this change to fix the broken import process, right? We can't assume that this is the correct fix, see [LPS-151143](https://issues.liferay.com/browse/LPS-151143). I know you're including this just to make sure it works and that this isn't your main fix, but just wanted to include this as a note because if you plan on submitting your fix to Master, we'll need to keep it in mind.